python/servo/testing_commands.py

```diff
@@ -215,14 +217,14 @@ def test_unit(self, build_type: BuildType, test_name=None, package=None, bench=F
             return 0
 
         # Gather Cargo build timings (https://doc.rust-lang.org/cargo/reference/timings.html).
-        args = ["--timings"]
+        args: list[str] = ["--timings"]
 
         if build_type.is_release():
             args += ["--release"]
         elif build_type.is_dev():
             pass  # there is no argument for debug
         else:
-            args += ["--profile", build_type.profile]
+            args += ["--profile", build_type.profile or ""]
```

Comment:
Hrm. I think this is an issue. Probably `profile` should be initialized to a reasonable default value instead of being possibly `None`.